client_plugin/drivers/shared/kvstore/etcdops/etcdops.go

```diff
@@ -521,13 +522,14 @@ func addrToEtcdClient(addr string) (*etcdClient.Client, error) {
 func (e *EtcdKVS) List(prefix string) ([]string, error) {
 	var keys []string
 
-	etcd := e.createEtcdClient()
-	if etcd == nil {
-		return nil, fmt.Errorf(etcdClientCreateError)
+	etcdAPI := e.createEtcdClient()
```

Comment:
It's because we already have the import package named as etcdClient:
`etcdClient "github.com/coreos/etcd/clientv3"`
I will rename the package to etcd and change etcdAPI to etcdClient.